### PR TITLE
checkers look at website

### PR DESCRIPTION
**Pull request by dneise**
initial proposal for FactIntervalCheckers, which look at the website and decide if to `warn` or `info`

### CODE REVIEW

File: shifthelper/checks.py [hunk 1]
```diff
@@ -1,53 +1,84 @@
 from custos import IntervalCheck
-import smart_fact_crawler
+import smart_fact_crawler as sfc
 from .tools.is_shift import is_shift_at_the_moment
+import requests
+from abc import ABCMeta, abstractmethod
```
Comment (relleums) [inline shifthelper/checks.py]: Why not using a base class until the very moment when 'NotImplementedError' is risen? The more restrictive ABC concept is not pythonic in my opinion. Further, it is one import more and might be less common than the std exception 'NotImplementedError'. I vote for rising 'NotImplementedError'.

Comment (dneise) [inline shifthelper/checks.py]: I learned, that `abstractmethod` is suitable when the API designer wants to express: "this thing is utterly useless without implementing this method".
Also it prevents from typos...

File: shifthelper/checks.py [hunk 2]
```diff
@@ -1,53 +1,84 @@
...
-        if is_shift_at_the_moment():
-            value = smart_fact_crawler.status()['Dim_Control']
-            if 'Running' not in value:
-                self.warning("Main.js is not running")
+import pandas as pd
+import datetime
+from datetime import timedelta, datetime
+import numpy as np
+
+class FactIntervalCheck(IntervalCheck, metaclass=ABCMeta):
```
Comment (dneise) [inline shifthelper/checks.py]: HTTPCamelCase you fool

File: shifthelper/checks.py [hunk 3]
```diff
@@ -1,53 +1,84 @@
...
 
-class HumidityCheck(IntervalCheck):
     def check(self):
         if is_shift_at_the_moment():
-            value = smart_fact_crawler.weather()['Humidity_in_Percent']
-            if value >= 98:
-                self.warning("Humidity > 98%")
+            text = self.inner_check()
+            if text is not None:
```
Comment (dneise) [inline shifthelper/checks.py]: if text?

File: shifthelper/checks.py [hunk 4]
```diff
@@ -1,53 +1,84 @@
...
-                self.warning("Wind speed > 50 km/h")
+    def all_recent_alerts_acknowledged(self):
+        all_alerts = requests.get('http://localhost:5000/alerts').json()
+        if not all_alerts:
+            return False
+        
+        now = datetime.utcnow()
+        all_alerts = pd.DataFrame(all_alerts)
+        all_alerts['timestamp'] = pd.to_datetime(all_alerts.timestamp, utc=True)
+
```
Comment (dneise) [inline shifthelper/checks.py]: pep8 too many empt lines in max head

**Pull request merged**